integration-tests/cache/pom.xml

```diff
@@ -67,6 +80,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
this would eventually go into `integration-tests/pom.xml`, right?

Reply:
If we move extension ITs into its own module, e.g. `integration-tests/extensions` we could configure it in one place. FYI @zakkak

Reply:
> this will eventually go into `integration-tests/pom.xml`, right?

Yes, that's the plan.

> If we move extension ITs into its own module, e.g. `integration-tests/extensions` we could configure it in one place. 

I am probably missing something, but I don't see any real harm letting the rule run for _all_ of `integration-tests`.

Reply:
Pulled up to `integration-tests` but level has to be switched from WARN to ERROR after every submodule was extended.